go.mod

```diff
@@ -19,7 +19,7 @@ require (
 	github.com/songgao/water v0.0.0-20200317203138-2b4b6d7c09d8
 	github.com/stretchr/testify v1.8.1
 	github.com/vishvananda/netlink v1.1.0
-	golang.org/x/net v0.0.0-20220630215102-69896b714898
+	golang.org/x/net v0.5.0
```

Comment:
@simaishi @rooftopcellist Do changes to these files break anything on your end? Unsure how the downstream golang builds work these days.

Reply:
We will need to re-create vendor tarball, but please go ahead and merge when you're ready. When a commit requires new vendor file, a pre-merge job for rpm build will fail and we'll know an update is needed.